packages/mobile/src/screens/notifications-screen/Notifications/TipReceivedNotification.tsx

```diff
@@ -29,19 +36,40 @@ const messages = {
   reactionSent: 'Reaction Sent!'
 }
 
+const useReaction = (signature: string) => {
+  return useSelectorWeb(makeGetReactionForSignature(signature))
```

Comment:
i think we want to memoize the makeGetReactionForSignature result so the selector doesn't see a new function coming in? not sure if that's a requirement for useSelector not triggering re-render tho....

Reply:
good call